### PR TITLE
PRX-1404: adding command line option '-b' for setting socket receive …

### PR DESCRIPTION
**Pull request by chris-heath**
…buffer size.

### CODE REVIEW

File: rtp_cluster.py [hunk 1]
```diff
@@ -121,6 +122,9 @@ def reopen(signum, logfile):
         if o == '-D':
             debug_threads = True
             continue
+        if o == 'b':
```
Comment (jnewman02) [inline rtp_cluster.py]: This should be '-b'.

File: rtp_cluster.py [hunk 2]
```diff
@@ -76,7 +76,7 @@ def reopen(signum, logfile):
     global_config = {}
 
     try:
-        opts, args = getopt.getopt(sys.argv[1:], 'fP:c:L:s:o:dD')
+        opts, args = getopt.getopt(sys.argv[1:], 'fP:c:L:s:o:dDb:')
```
Comment (NormB) [inline rtp_cluster.py:79]: The colon (:) after the b means that it is a "required" parameter.  Is this correct ?

**Review by NormB — APPROVED** [review]

File: rtp_cluster.py (hunk 2 above)
Comment (chris-heath) [reply rtp_cluster.py:79]: @NormB Actually the colon signifies options that "require" an argument/value.  So if you specify the '-b' option, you have to pass in the appropriate value with it.

**Review by jnewman02 — COMMENTED** [review]
Great work, looks good, approving.

**Pull request merged**